man/tpm2_quote.1.md

```diff
@@ -41,18 +41,25 @@
 
   * **-m**, **--message**:
 
-    message output file, records the quote message that makes up the data that
+    Message output file, records the quote message that makes up the data that
```

Comment:
separate patch for these to fixups in tpm2_quote.md.1

Reply:
Should this be a separate PR or just in its own patch/commit?

Reply:
Its own commit on this PR is 100% OK.